TWLight/applications/views.py

```diff
@@ -809,13 +811,23 @@ class ListReadyApplicationsView(CoordinatorsOnly, ListView):
     template_name = 'applications/send.html'
 
     def get_queryset(self):
+        # The annotate and filter functions filter out any partners who only
```

Comment:
This warrants a bit more explanation/commentary. Please make this slightly more human parse-able even if it makes the code somewhat less concise.